robocopkaka/bootcamp-29-project
# #158186873 Fix admin profile component

#### What does this PR do?
fix admin profile component
#### Description of Task to be completed?
* allow admins edit events and centers using a modal
* add pagination links to the admin profile page for both events and centers
* write tests for the AdminProfile component
#### How should this be manually tested?
* clone this repo
* cd into the project folder
* run `npm install` to install all dependecies
* run `npm run start:all` to start up the application
* go to `localhost:8000` on your browser
* click on `Signup`
* enter your details
* click on `Signup`
* click on `Login`
* enter your details
* click on `Login`
* click on the `Profile` dropdown
* click on `Profile`
* you should see all events and centers accessible via tabs
#### Any background context you want to provide?
haven't written end to end tests for this feature
#### What are the relevant pivotal tracker stories?(if applicable)
#158186873
